NetCasbin.UnitTest/EnforcerUnitTest.cs

```diff
@@ -83,52 +83,52 @@ public async Task TestKeyMatchModelInMemoryAsync()
 
             var e = new Enforcer(m, a);
 
-            TestEnforce(e, "alice", "/alice_data/resource1", "GET", true);
-            TestEnforce(e, "alice", "/alice_data/resource1", "POST", true);
-            TestEnforce(e, "alice", "/alice_data/resource2", "GET", true);
-            TestEnforce(e, "alice", "/alice_data/resource2", "POST", false);
-            TestEnforce(e, "alice", "/bob_data/resource1", "GET", false);
-            TestEnforce(e, "alice", "/bob_data/resource1", "POST", false);
-            TestEnforce(e, "alice", "/bob_data/resource2", "GET", false);
-            TestEnforce(e, "alice", "/bob_data/resource2", "POST", false);
-
-            TestEnforce(e, "bob", "/alice_data/resource1", "GET", false);
-            TestEnforce(e, "bob", "/alice_data/resource1", "POST", false);
-            TestEnforce(e, "bob", "/alice_data/resource2", "GET", true);
-            TestEnforce(e, "bob", "/alice_data/resource2", "POST", false);
-            TestEnforce(e, "bob", "/bob_data/resource1", "GET", false);
-            TestEnforce(e, "bob", "/bob_data/resource1", "POST", true);
-            TestEnforce(e, "bob", "/bob_data/resource2", "GET", false);
-            TestEnforce(e, "bob", "/bob_data/resource2", "POST", true);
-
-            TestEnforce(e, "cathy", "/cathy_data", "GET", true);
-            TestEnforce(e, "cathy", "/cathy_data", "POST", true);
-            TestEnforce(e, "cathy", "/cathy_data", "DELETE", false);
+            await TestEnforceAsync(e, "alice", "/alice_data/resource1", "GET", true);
```

Comment:
I found some lint warnings in this file, would be great to fix in this pr.
![图片](https://user-images.githubusercontent.com/4054836/94510765-25350800-024a-11eb-82de-2370bae589a7.png)

Reply:
I know these lint warnings. But it does not have an actual impact for the user and Because it is not related and I think it should not be a part of this pull request. I will fix these at other pull requests or commit.